expo/gbe/templates/gbe/assign_volunteer.tmpl

```diff
@@ -129,9 +129,9 @@
             </td>
             <td class="bid-table">
               {% if event_window.window %}
-                {{event_window.window.day.day|date:"D"}},
-                {{event_window.window.start|date:"f A"}} -
-                {{event_window.window.end|date:"f A"}}
+                {{event_window.window.day.day|date:_("DATE_FORMAT")}},
+                {{event_window.window.start|date:_("DAY_FORMAT DATE_FORMAT") }} -
+                {{event_window.window.end|date:_("DAY_FORMAT DATE_FORMAT")}}
               {% endif %}
```

Comment:
I'm seeing a problem here... 

1 - use recent live data
2 - turn on GBE2015 as an upcoming conference
3 - Go to:

http://localhost:8282/volunteer/assign/640

This is my volunteer bid from last year, when we had many events and time windows to assign to...

Under the column "time window"  all windows render as "Feb. 5, 2016, - "

When they should be "Feb. 5, 2016, HH:MM - HH:MM"

This is a blocker, it makes the time windows unreadable when assigning volunteers.